airbyte-integrations/connectors/destination-sftp-json/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.0"
+name = "destination_sftp_json"
+description = "Destination implementation for Sftp Json."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/destinations/sftp-json"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+[[tool.poetry.packages]]
+include = "destination_sftp_json"
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+paramiko = "==2.7.2"
+smart-open = "==5.1.0"
+airbyte-cdk = "==0.1.36"
+
+[tool.poetry.scripts]
+destination-sftp-json = "destination_sftp_json.run:run"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^6.1"
+docker = "==5.0.3"
```

Comment:
The new docker library is 7+, but there are cross-dependencies. It could be that it's better to migrate first and upgrade separately, this is rather high usage.